.github/workflows/test.yml

```diff
@@ -66,3 +66,31 @@ jobs:
       # Cleanup cached paths
       - run: tarantoolctl rocks remove cartridge
       - run: tarantoolctl rocks remove ddl
+  benchmark:
+    strategy:
+      fail-fast: false
+      matrix:
+        tarantool: ['1.10', '2.5', '2.6', '2.7']
+        coveralls: [false]
+        include:
+          - tarantool: '2.8'
+            coveralls: true
```

Comment:
This is dead code (and I think that we don't need to collect coverage on the benchmark).

Reply:
deleted